tests/pytests/integration/runners/test_saltutil.py

```diff
@@ -98,6 +98,36 @@ def world():
         assert "{}.hello".format(module_type) in ret.stdout
 
 
+def test_sync_refresh_false(
+    module_type, module_sync_functions, salt_run_cli, salt_minion, salt_master
+):
+    """
+    Ensure modules are synced when various sync functions are called
+    """
+    module_name = "hello_sync_{}".format(module_type)
```

Comment:
shouldn't we be moving to f strings now?

Reply:
will fix it